src/env.h

```diff
@@ -406,6 +406,7 @@ constexpr size_t kFsStatsBufferLength =
   V(filehandlereadwrap_template, v8::ObjectTemplate)                           \
   V(fsreqpromise_constructor_template, v8::ObjectTemplate)                     \
   V(handle_wrap_ctor_template, v8::FunctionTemplate)                           \
+  V(histogram_ctor_template, v8::ObjectTemplate)                               \
```

Comment:
The naming in a lot of the pre-existing cases is just as unfortunate, but as this is the instance template rather than the constructor function template, I’d prefer to avoid the term `ctor` :)

Reply:
renamed :-)